build/clean-up-cluster.sh

```diff
@@ -3,20 +3,57 @@
 
 
 set -e
-UI_CURRENT_IMAGE=$1
 
-make docker/login
-export DOCKER_URI=quay.io/stolostron/grc-ui-tests:latest
-make docker/pull
+function hub() {
+    echo "Hub: clean up"
+    oc delete ns duplicatetest || true
```

Comment:
Instead of `|| true`, could you use the `--ignore-not-found` flag instead?

Reply:
Great. Didn't know the flag exist. Will update